### PR TITLE
Document private mailing list for GitHub Admin team

### PR DESCRIPTION
**Pull request by nikhita**
The mailing list has been already setup.

/cc @kubernetes/owners 
/assign @cblecker @spiffxp

### CODE REVIEW

File: github-management/opening-a-request.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Please open an issue against the [kubernetes/test-infra] repository describing
 your issue. If your request is urgent, please escalate to the
 [test-infra on-call] or reach out to `#testing-ops` on Slack.
 
+_To report any sensitive information, please email the private github@kubernetes.io list_.
```
Comment (mrbobbytables) [inline github-management/opening-a-request.md]: Possible formatting suggestion/nit --  With this directly under the bot/automation header it kind of looks like its a part of that section. Maybe another header or a horizontal rule just to break it up?

Comment (nikhita) [reply github-management/opening-a-request.md]: I thought it felt awkward too. Updated, ptal if this is any better!

Comment (mrbobbytables) [reply github-management/opening-a-request.md]: Thanks! That looks better.

**Comment by spiffxp** [comment]
/lgtm
I leave the /approve to @cblecker

**Comment by cblecker** [comment]
/lgtm
/approve

Thanks @nikhita

**Pull request merged**